docs/pyrona.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: default
+title: Pyrona - Fearless Concurrency for Python
+---
+
+# Pyrona - Fearless Concurrency for Python
+
+As part of Project Verona, we have been developing a new ownership model for Python, called Lungfish.
+This model is designed to provide a safe and efficient way to manage memory and concurrency in Python programs.
+Pyrona is our experiments to develop this approach.
+
+
+## Plan for Python
+
+### Quick prototyping in FrankenScript
+
+Modifying a production language is a complex task.
+As an initial step, we have been developing a toy language called [FrankenScript](https://github.com/fxpl/frankenscript) that allows us to quickly prototype our ideas for region-based ownership.
+FrankenScript is a small language that is designed to be easy to modify and extend.
+It is based on the ideas of ownership and concurrency that we are exploring in Project Verona, but where all the checks are dynamic.
+
+This prototype has given us confidence in the conceptual ideas behind our ownership model, and has allowed us to explore the design space of ownership in a dynamic language.
```

Comment:
```suggestion
This prototype has given us confidence in the conceptual ideas behind our ownership model, and has allowed us to explore the design space of ownership in a dynamic language. It has also helped us explain our ideas and the different design decisions to external people.
```

Not sure about "external people" -- don't know what the right term is. Stakeholder seems wrong. Members of the Python community perhaps?

Reply:
Yeah, that is a bad phrase.  "...to people outside the Project Verona team."

Reply:
Perfect!